src/bootstrap.ts

```diff
@@ -176,6 +179,11 @@ export const run = async (config: AppConfig) => {
       throw new Error('App is not configured correctly either provide jwt pub key url or key');
     }
     setJwtPublicKey(config.jwtPubKeyUrl());
+    setInterval(() => {
```

Comment:
does the public key change every 5min now??

Reply:
No this is just something should been done before to adapt automatically without manual restarts

Reply:
I see makes sense